aptos-move/aptos-vm/src/move_vm_ext/vm.rs

```diff
@@ -191,6 +191,15 @@ impl MoveVmExt {
         &self,
         resolver: &'r S,
         session_id: SessionId,
+    ) -> SessionExt<'r, '_> {
+        self.new_session_with_flush_flag(resolver, session_id, false)
+    }
+
+    pub fn new_session_with_flush_flag<'r, S: AptosMoveResolver>(
```

Comment:
Maybe add a comment why this flag is useful? Otherwise we have different APIs for sessions which are created differently. Also, what you can do instead is to have ` new_clean_session` which has the implementation with `must_flush = true`. This way you can make this function debugger-only (mark as a cfg feature?)

Reply:
So having a private `new_session_impl` and then wrapping `new_session` and `new_clean_session` seems cleaner than having one public API call some other public API?

Reply:
done